src/components/navigation.tsx

```diff
@@ -15,10 +22,18 @@ interface Props {
     faq: string;
     contact: string;
   };
+  languageLabels: {
```

Comment:
Are the 'languages' the same as the 'locale'? Or are these different types of information? If the former, perhaps rename `languageLabels` to `localeLabels`?